src/main/java/com/google/devtools/build/lib/analysis/test/TestStrategy.java

```diff
@@ -260,9 +261,9 @@ private static void addRunUnderArgs(TestRunnerAction testAction, List<String> ar
         args.add("\"$@\"");
         args.add(shellExecutable); // Sets $0.
       }
-      args.add(execSettings.getRunUnder().getCommand());
+      args.add(((CommandRunUnder) execSettings.getRunUnder()).command());
```

Comment:
Should we also change line 246 (`if (execSettings.getRunUnderExecutable() != null)`) to instead be an `instanceof LabelRunUnder` check, to make it more obvious why this cast is safe?

Reply:
Done (with a switch since the `else` branch wouldn't automatically cast to a `CommandRunUnder`).